bitnami/postgresql-ha/templates/_helpers.tpl

```diff
@@ -734,3 +735,12 @@ postgresql-ha: Upgrade repmgr extension
       --set postgresql.upgradeRepmgrExtension=true
 {{- end -}}
 {{- end -}}
+
+{{/* Set PGPASSWORD as environment variable depends on configuration */}}
+{{- define "postgresql-ha.pgpassword" -}}
+{{- if .Values.postgresql.usePasswordFile -}}
+PGPASSWORD=$(cat $POSTGRES_PASSWORD_FILE)
+{{- else -}}
+PGPASSWORD=$POSTGRES_PASSWORD
+{{- end -}}
```

Comment:
I'm not sure about this depending on environment variables. Can we make it only dependant on the values itself?